app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+  has_many :recipe
+  has_many :food
+end
```

Comment:
- [ ] Hi Justice, great job so far :muscle:  there is just a tiny change to make here. Kindly add the pluralized versions of the models in this case as suggested in the [rails documentation](https://guides.rubyonrails.org/association_basics.html) to correctly set the associations :wink: